packages/dicomImageLoader/.webpack/webpack-dynamic-import.js

```diff
@@ -6,7 +6,7 @@ const TerserPlugin = require('terser-webpack-plugin');
 const outputPath = path.join(rootPath, 'dist', 'dynamic-import');
 
 const prodConfig = {
-  mode: 'production',
+  mode: 'development',
```

Comment:
revert this file please

Reply:
I can revert this line, but not the file.  Going to do that late when things are nearly ready.